src/js/structs/UniversePackage.js

```diff
@@ -68,7 +68,13 @@ class UniversePackage extends Item {
   }
 
   getPreInstallNotes() {
-    return this.get("preInstallNotes");
+    // punctuate the end if not present
```

Comment:
I'm adding this because there are some packages (for example [cassandra](https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/C/cassandra/29/package.json#L19)) don't end the preinstall notes with a period. The proper fix would be for the package itself to not do this, but I think it's worth the patch in the UI code. If I don't add this, then when appending the T&C it lacks a period, which is not nice.

Reply:
It feels like this should be inside a util instead the struct

Reply:
Yeah true, I can move into string util.